todo-backend/TodoBackend/function.js

```diff
@@ -0,0 +1,82 @@
+const rp = require('request-promise-native');
+
+const execute = async (command, field, value) => {
+  const request = {
+    command: command,
+    key: 'TodoBackend',
+    field: field,
+    value: JSON.stringify(value)
+  };
+  const options = {
+    url: `https://run-sandbox.binaris.com/v2/run/${process.env.BINARIS_ACCOUNT_NUMBER}/redis_todobackend`,
```

Comment:
by including the binaris sdk you can remove the need to hardcode the url. Take a look at this example line 24 https://github.com/binaris/functions-examples/blob/master/quicksearch/functions/util.js

Reply:
Hardcoded URLs are the way described in the docs. Should we update the docs with the SDK?
Again, I want to keep this example in sync with the tutorial.

Reply:
yes, like I mentioned above this is ending up being an advanced tutorial. I will send you a separate list of environment variables available in the SDK that we should document.